includes/admin/amp-wp-admin-styles.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+add_action('admin_head', 'amp_options_styles');
+function amp_options_styles() {
```

Comment:
Hmm. I replaced the action added to admin_head with this one and the styles do not get applied.